presto-tpch/src/main/java/com/facebook/presto/tpch/TpchRecordSet.java

```diff
@@ -198,14 +197,6 @@ public void close()
             closed = true;
         }
 
-        private boolean rowMatchesPredicate()
```

Comment:
you remove this method only to add it later on? Are the commits ordered properly?